### PR TITLE
ci: enable cache and multithreading for GA

### PR DESCRIPTION
**Pull request by orzyyyy**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    enable cache for GA       |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by orzyyyy** [comment]
TimeStamp: circleci finished.

**Comment by orzyyyy** [comment]
TimeStamp: github action finished.

**Comment by orzyyyy** [comment]
TimeStamp: start

**Comment by orzyyyy** [comment]
GA faster than circleci somtimes, but somttimes not. Need more cases.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -3,16 +3,217 @@ name: test
...
+      - name: restore cache from lib
+        uses: actions/cache@v1
+        with:
+          path: lib
+          key: lib-${{ github.sha }}
+
+      - name: test
+        run: npm test
+        env:
+          LIB_DIR: lib
```
Comment (shaodahong) [inline .github/workflows/test.yml]: 这些和 travis 重复了，

Comment (afc163) [reply .github/workflows/test.yml]: travis 可删

Comment (shaodahong) [reply .github/workflows/test.yml]: github action 也不是很稳定，至少我看到挂了好几次

Comment (orzyyyy) [reply .github/workflows/test.yml]: 可以并行多看看嘛，反正是白嫖。我是想看看缓存的 GA 能不能稍微赶上不加缓存的 circieci。

**Comment by zombieJ** [comment]
https://github.com/ant-design/ant-design/runs/318684879

这个错误需要看一下

**Pull request merged**